website/content/en/status/report-2023-04-2023-06/mfsbsd-building-integration.adoc

```diff
@@ -0,0 +1,28 @@
+=== Integrate mfsBSD into the release building tools
+
+Links: +
+link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[Wiki Article] URL: link:https://wiki.freebsd.org/SummerOfCode2023Projects/IntegrateMfsBSDIntoTheReleaseBuildingTools[]
+
+Contact: Soobin Rho <soobinrho@FreeBSD.org>
+
+==== What is mfsBSD?
+
+"link:https://people.freebsd.org/~mm/mfsbsd/mfsbsd.pdf[mfsBSD] is a toolset to create small-sized but full-featured mfsroot based distributions of FreeBSD that store all files in memory (MFS) [Memory File System] and load from hard drive, usb storage device or optical media.
+It can be used for a variety of purposes, including diskless systems, recovery partitions and remotely overwriting other operating systems."
+
+mailto:mm@FreeBSD.org[Martin Matuska] is both the author of the white paper above and the maintainer of the link:https://github.com/mmatuska/mfsbsd[mfsBSD repository].
```

Comment:
How about this?  ...is both the author of the link:https://people.freebsd.org/~mm/mfsbsd/mfsbsd.pdf[white paper] and the...

I know you already link to the white paper above, but it took me a moment to realize what you meant by saying the "white paper *above*".  Another option could be to say "is both the author of the white paper linked above...".

Reply:
Other than this optional change it looks good to me.

Reply:
Absolutely! That makes the writing much nicer. Changed to `link:https://people.freebsd.org/~mm/mfsbsd/mfsbsd.pdf[white paper]` now. Thanks @Jehops!